packages/card/src/react/__specs__/index.spec.tsx

```diff
@@ -3,8 +3,8 @@ import { screen, render } from '@testing-library/react'
 import { axe } from 'jest-axe'
 import React from 'react'
 
-import Card from '../index'
 import * as stories from '../__stories__/index.story'
+import Card from '..'
```

Comment:
why?

Reply:
At this point the main reason is for codebase consistency. Currently, every other relative index import uses this syntax. 

Previously, as a part of the esm refactor, when we were going for true ES modules, the adjustment was made for the mandatory file extension https://nodejs.org/api/esm.html#esm_mandatory_file_extensions . I know we pulled a little back from that level of fidelity to the standard since then, and that's fine, but we did not remove this pattern.